### PR TITLE
Update INTRODUCTION.md

### PR DESCRIPTION
**Pull request by rafinskipg**
Added a custom Theming section

### CODE REVIEW

File: INTRODUCTION.md [hunk 1]
```diff
@@ -105,6 +105,33 @@ const PostForm = ({model}) =>
 
 That's all! `AutoForm` will generate a complete form with labelled fields, errors list (if any) and a submit button. Also, it will take care of validation and handle model changes.
 
+If you want to use custom HTML code inside your form you can include the form content.
+
+```javascript
+// Choose your theme
+import AutoForm from 'uniforms-unstyled/AutoForm';
```
Comment (janowsiany) [inline INTRODUCTION.md]: Please, keep imports sorted.

File: INTRODUCTION.md [hunk 2]
```diff
@@ -105,6 +105,33 @@ const PostForm = ({model}) =>
...
+import AutoField from 'uniforms-unstyled/AutoField';
+import TextField from 'uniforms-unstyled/TextField';
+import SubmitField from 'uniforms-unstyled/SubmitField';
+
+// A compatible schema
+import PostSchema from './schemas/Post';
+
+const PostForm = ({model}) =>
+    (<div>
```
Comment (janowsiany) [inline INTRODUCTION.md]: Unnecessary HTML markup and please keep this formatting:
```js
const PostForm = ({model}) => (
  //code
);
```

File: INTRODUCTION.md [hunk 3]
```diff
@@ -105,6 +105,33 @@ const PostForm = ({model}) =>
...
+            <h2>Title</h2>
+            <AutoField name="myField"/>
+            <TextField name="otherField" />
+            <div className="pull-right">
+              <SubmitField className="btn btn-primary"/>
+            </div>
+        </AutoForm>
+    </div>)
+;
+
```
Comment (janowsiany) [inline INTRODUCTION.md]: Remove not needed spacing.

File: INTRODUCTION.md [hunk 4]
```diff
@@ -105,6 +105,33 @@ const PostForm = ({model}) =>
...
+import SubmitField from 'uniforms-unstyled/SubmitField';
+
+// A compatible schema
+import PostSchema from './schemas/Post';
+
+const PostForm = ({model}) =>
+    (<div>
+        <AutoForm schema={PostSchema} onSubmit={doc => db.save(doc)} model={model}>
+            <h2>Title</h2>
+            <AutoField name="myField"/>
```
Comment (janowsiany) [inline INTRODUCTION.md]: Fix eslint errors `react/jsx-space-before-closing` here in other places.

**Comment by radekmie** [comment]
@rafinskipg?

**Pull request merged**